build-model

```diff
@@ -3,6 +3,7 @@ set -e -v
 
 # regenerate the fabric8 java API including k8s custom resources
 pushd $(go env GOPATH)/src/github.com/fabric8io/kubernetes-model/
-git checkout sk8s
+git fetch origin
+git checkout origin/sk8s-sidecar-scaling
```

Comment:
can we go ahead and push the revised model so that this PR doesn't need to change the branch here from "sk8s"? (seems possible since the only change is the addition of "replicase" for Topic resources, and that is not a required property since it defaults to 1)